LX/src/main/java/heronarts/lx/LXSnapshotActivePattern.java

```diff
@@ -0,0 +1,39 @@
+package heronarts.lx;
+
+import com.google.gson.JsonObject;
+
+public class LXSnapshotActivePattern extends LXComponent {
```

Comment:
I might consider making this a private inner class of LXSnapshot. And I'm not really sure if this ought to extend LXComponent. Would just be inside LXSnapshot and look like:

private class ActivePattern {
}

Reply:
Agreed.  It was a more complicated class at one point before I changed it to saving just the indexes.